.gitignore

```diff
@@ -0,0 +1,31 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+node_modules/
+# Ignore all logfiles and tempfiles.
```

Comment:
- [ ] As requested by the previous reviewer, the `node modules` folder should not be pushed to github. as it was already pushed it, I recommended you to remove it first from github, put it in gitignore and push again. thank yo

![Screenshot from 2023-02-24 15-05-23](https://user-images.githubusercontent.com/65860678/221187099-2a33f55e-2c10-44b7-b442-c7b5fb5d00b7.png)